src/main/java/uk/gov/hmcts/reform/iacaseaccessapi/infrastructure/config/SwaggerConfiguration.java

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
@colmlynch60 Any reason to comment out this entire file

Reply:
Apologies - I removed it when upgrading swagger dependency. I've replaced this configuration file with OpenApiConfiguration and same as other IAC services.